middleware/categories/index.js

```diff
@@ -0,0 +1,18 @@
+const { check, validationResult } = require('express-validator');
```

Comment:
Recomiendo que la carpeta se llame middlewares, en plural

Reply:
Tiene sentido crear un index?, si van a haber middlewares para categorias, crearía solo el archivo categoriesMiddlewares

Reply:
**refactor:** middlewares name and file categoriesMiddlewares
![imagen](https://user-images.githubusercontent.com/75862846/159955726-f5e08abc-711b-40ad-8ebe-d397ae0e54fd.png)